### ISSUE TITLE
EPUB-Checker not starting on macOS 11.0 "Big Sur" - Java not found

### ISSUE THREAD

**Issue by tofi86** [paginagmbh/EPUB-Checker#67]
Java is not found on macOS 11.0 "Big Sur":

![Bildschirmfoto 2021-01-06 um 16 03 24](https://user-images.githubusercontent.com/365118/103783360-e7116680-5038-11eb-932f-3e287fdfad3d.png)

even though a JDK is installed and registered:

```
$ /usr/libexec/java_home -V
Matching Java Virtual Machines (1):
    1.8.0_275 (x86_64) "AdoptOpenJDK" - "AdoptOpenJDK (JRE) 8" /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jre/Contents/Home
```

That's a `universalJavaApplicationStub` issue (https://github.com/tofi86/universalJavaApplicationStub/issues/91) on macOS 11.0 which has been resolved recently. Once the new version 3.1.0 is released, it must be included in the EPUB-Checker builds.

**Comment by tofi86** [comment]
`universalJavaApplicationStub` v3.1.0 has been released with support for macOS 11.0 "Big Sur"

https://github.com/tofi86/universalJavaApplicationStub/releases/tag/v3.1.0

**Comment by tofi86** [comment]
Fixed with 6d991f6332fa838a0f0cad1c527276cc3380a8ae

However, the macOS 11.0 issue regarding the launcher stub compilation (#66) is still unresolved...
